src/js/models/v2/awards/CoreAward.js

```diff
@@ -11,7 +11,7 @@ export const formatDate = (date) => date.format('MM/DD/YYYY');
 
 const CoreAward = {
     populateCore(data) {
-        this._category = data.category || 'idv'; // IDVs have null values
+        this._category = data.category;
```

Comment:
There  aren't many left, but 2k awards in staging have no type nor a category. Probably want to add `|| '';`

Reply:
👍 Null categories will be parsed as Financial Assistance, is that ok?

Reply:
A majority of those records are FABS, so that is acceptable behavior.